requirements.in

```diff
@@ -2,15 +2,15 @@ apscheduler~=3.6
 Click~=7.0
 cachetools~=3.1
 dictdiffer
-gunicorn~=20.0.4
+gunicorn~=20.0
 h5py~=2.8
 influxdb~=5.2
 jinja2~=2.10
 pyarrow~=0.15
 numpy~=1.17
 pandas~=0.25
 peewee~=3.12
-numexpr>=2.6.1
+numexpr>=2.6
```

Comment:
I wonder if we have a conscious and predictable approach to using compatible (~=) vs greaterequal (>=). To me, it feels sensible to use greater than unless we are "afraid" greater than will break our stuff, makes it easier to live together with other requirements (?). But I don't know. Maybe someone else have opinions? @epa095 @milesgranger @ryanjdillon

Reply:
I prefer compatible with *as the general rule*. It only really makes a difference with major versions (thats where they differ), and with those I think it makes sense that we by default dont assume that we are compatible with newer major versions.

Reply:
It does appear from looking at things like [sklearn's setup.py](https://github.com/scikit-learn/scikit-learn/blob/f4da713c44d9529da2d82d76e0c0c6ef7e0e4795/setup.py#L260) as well as [Dask's](https://github.com/dask/dask/blob/3ba4f6e43ecda75ed4ad19f42f3eba6c78a42d49/setup.py#L23), [requests](https://github.com/psf/requests/blob/eedd67462819f8dbf8c1c32e77f9070606605231/setup.py#L44) and others, that using `>=` seems to be the norm, even with requirements which aren't at a major version. :man_shrugging: